### PR TITLE
Add release notes for 1.26.3

### PR DESCRIPTION
**Pull request by jrhee17**
스크린샷 2023-11-17 오전 11 39 00

### CODE REVIEW

**Review by minwoox — APPROVED** [review]
Thanks! 🙇

File: site/src/pages/release-notes/1.26.3.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+date: 2023-11-16
+---
+
+
```
Comment (minwoox) [inline site/src/pages/release-notes/1.26.3.mdx:5]:
```suggestion
```

**Review by ikhoon — APPROVED** [review]
Thanks!

File: site/src/pages/release-notes/1.26.3.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Updated the GraalVM `native-image` configuration. #5296 #5308
+
+## 🙇 Thank you
+
+<ThankYou usernames={[
+  'ikhoon',
+  'jrhee17',
+  'minwoox',
+  'trustin',
+  'yswdqz'
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.26.3.mdx:19]:
```suggestion
  'trustin'
```

**Review by ikhoon — APPROVED** [review]
Still LGTM

File: site/src/pages/release-notes/1.26.3.mdx (hunk 2 above)
Comment (jrhee17) [reply site/src/pages/release-notes/1.26.3.mdx:19]: I believe this is the reporter of the issue https://github.com/line/armeria/issues/5296 which I think is worth crediting

**Pull request merged**